### PR TITLE
Enabling support in the geekbench workload for version 5.0.0 of the c…

### PR DESCRIPTION
**Pull request by scojac01**
…orporate workload

### CODE REVIEW

File: wa/workloads/geekbench/__init__.py [hunk 1]
```diff
@@ -29,34 +29,25 @@ class Geekbench(ApkUiautoWorkload):
...
     Geekbench benchmarks: http://www.primatelabs.com/geekbench/doc/benchmarks.html
-
     Geekbench scoring methedology:
     http://support.primatelabs.com/kb/geekbench/interpreting-geekbench-scores
-
     """
     summary_metrics = ['score', 'multicore_score']
     versions = {
-        '4.0.1': {
+        '4.2.0': {
```
Comment (marcbonnici) [inline wa/workloads/geekbench/__init__.py]: Can we not have support for both `4.0.1` and `4.2.0`?
Also this maybe should be in a separate commit?

File: wa/workloads/geekbench/uiauto/app/src/main/java/com/arm/wa/uiauto/geekbench/UiAutomation.java [hunk 2]
```diff
@@ -184,4 +190,10 @@ public void shareResults() throws Exception {
         shareButton.waitForExists(WAIT_TIMEOUT_5SEC);
         shareButton.click();
     }
+
+    public void scrollPage() throws Exception {
+        UiScrollable listView = new UiScrollable(new UiSelector().className("android.widget.ScrollView"));
+        listView.setMaxSearchSwipes(3);
+        listView.scrollTextIntoView("RUN CPU BENCHMARK");
```
Comment (marcbonnici) [inline wa/workloads/geekbench/uiauto/app/src/main/java/com/arm/wa/uiauto/geekbench/UiAutomation.java]: Will the list view also be present in geekbench corporate 4? Or should this check for the existence of the scroll view first?

File: wa/workloads/geekbench/__init__.py (hunk 1 above)
Comment (scojac01) [reply wa/workloads/geekbench/__init__.py]: So do you want me to just add another entry for 4.2.0?

File: wa/workloads/geekbench/__init__.py [hunk 3]
```diff
@@ -194,12 +186,13 @@ def update_result_4(self, context):
                     context.add_metric(namemify(section['name'] + '_' + workload_name + '_score', i),
                                        workloads['score'])
 
+    def update_result_5(self, context):
+        self.update_result_4
```
[the remark this answers is not in the document]
Comment (scojac01) [reply wa/workloads/geekbench/__init__.py]: I have changed this now and tested so will be updating soon.

File: wa/workloads/geekbench/uiauto/app/src/main/java/com/arm/wa/uiauto/geekbench/UiAutomation.java (hunk 2 above)
Comment (scojac01) [reply wa/workloads/geekbench/uiauto/app/src/main/java/com/arm/wa/uiauto/geekbench/UiAutomation.java]: list view is present in both.
